docs/man_pages/project/testing/debug-android.md

```diff
@@ -23,7 +23,7 @@ Debugs your project on a connected device, in a native emulator or in Genymotion
 * `--stop` - Detaches the debug tools.
 * `--get-port` - Retrieves the port on which you are debugging your application.
 * `--debug-port` - Sets a new port on which to attach the debug tools.
-* `--timeout` - Sets the number of seconds that NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 90 seconds.
+* `--timeout` - Sets the number of seconds the NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 90 seconds.
```

Comment:
that the 

You can keep it as it is but the additional "that" might bring more clarity for non-native English speakers.